package.json

```
@@ -12,7 +12,11 @@
   },
   "devDependencies": {
     "@stencil/core": "^2.0.0",
-    "@stencil/router": "^1.0.1"
+    "@stencil/router": "^1.0.1",
+    "@types/jest": "^26.0.24",
+    "jest": "^26.6.3",
```

Ah, it looks like we need to add an explicit dependency on `jest-cli` too - I thought that wouldn't be needed as it's a subpackage of `jest`, my mistake! Would you mind adding one more dependency?
```suggestion
    "jest": "^26.6.3",
    "jest-cli": "^26.6.3",
```